src/guild/Connection.ts

```diff
@@ -226,6 +226,14 @@ export class Connection extends EventEmitter {
         this.debug(`[Voice] <- [Discord] : Server Update Received | Server: ${this.region} Guild: ${this.guildId}`);
     }
 
+    /**
```

Comment:
I'm not sure about this change, as disconnect() will supply its own method to this.send, instead of having them use this function all together for simplicity